src/components/navigation/SlideMenu.tsx

```diff
@@ -1,156 +1,69 @@
-import React, { useMemo, useState } from 'react';
+import React, { ReactNode, useCallback, useMemo } from 'react';
```

Comment:
looks like we use `React.ReactNode` below, can we remove the import of ReactNode? I also prefer `React.ReactNode` anyway.

Reply:
why wasn't this caught by our linter? the import plugin usually spots this.

Reply:
It's only a warning when reported by the typescript-eslint rules. I've updated the linting config to include unused imports and setup a CI check for linting https://github.com/resident51/resident51/pull/3/commits/f379bf7c919f932e035213b5b0944921a6ffd8ad

Reply:
neat, and plus one for making our github actions actually do something